general.xml

```diff
@@ -3055,6 +3064,20 @@ Fil pilote > Off=0001 - On=0002</description>
 			</client>
 		</cluster>
 
+		<!-- IMMAX -->
+		<cluster id="0xfc82" name="Immax" mfcode="0x1037">
+			<description>Immax specific attributes.</description>
+			<server>
+				<attribute id="0x0010" name="Turned on since" type="u32" mfcode="0x1037" access="r" required="m"> </attribute>

```

Comment:
Cool.  Want one!

Reply:
:) Not to bad that you can see how log it has been turned on.

Hace silently hoped for something that can turn it off as well automatically.

Reply:
That’s standard _On/Off_ cluster functionality.  Try a PUT to the `state` with a body of `{"on": true, "ontime": 60}`.  Not sure if that’s in seconds or 0.1 seconds, though.  This sends the _On with Timed Off_ command (also used by the Trådfri motion sensors).

Reply:
Yeah, tend to always overlook that, but I meant based on current power usage. Off after 5 mins < 5W.